backend/src/src-cvesearch/src/main/java/org/eclipse/sw360/cvesearch/datasource/heuristics/SearchLevels.java

```diff
@@ -77,48 +93,49 @@ private void setup(CveSearchApi cveSearchApi, int vendorThreshold, int productTh
 
     public Stream<List<NeedleWithMeta>> apply(Release release) throws IOException {
         try {
-        return searchLevels.stream()
-                .map(searchLevel -> {
-                    try {
-                        return searchLevel.apply(release);
-                    } catch (IOException e) {
-                        throw new UncheckedIOException(e);
-                    }
-                }) ;
+            return searchLevels.stream()
+                    .map(searchLevel -> {
+                        try {
+                            return searchLevel.apply(release);
+                        } catch (IOException e) {
+                            throw new UncheckedIOException(e);
+                        }
+                    });
         } catch (UncheckedIOException ue) {
             throw ue.getIOExceptionCause();
         }
     }
 
-    class UncheckedIOException extends RuntimeException{
+    class UncheckedIOException extends RuntimeException {
```

Comment:
Here and at multiple other places in this PR: unnecessary change in formatting. We have agreed not to do that. See https://github.com/sw360/sw360portal/wiki/Dev-DoD-and-Style

Please correct this first as it makes very hard to see what has been actually changed in the functionality

Reply:
I have corrected the feature in my IDE, only changed lines will be formatted. I don't see the point to revert the line that were changed by mistake back to an inconsistant formatting.

Reply:
Thing is, it's not "the line". It's a lot of lines. I'm having a hard time reviewing what you actually changed. I can't and won't make you do it, but it would be oh so kind of you if you did.

Reply:
I reverted most of them, it should be more readable now :)